smallrye-reactive-messaging-kafka/src/main/java/io/smallrye/reactive/messaging/kafka/i18n/KafkaLogging.java

```diff
@@ -217,4 +217,8 @@
     @LogMessage(level = Logger.Level.ERROR)
     @Message(id = 18249, value = "Unable to recover from the deserialization failure (topic: %s), configure a DeserializationFailureHandler to recover from errors.")
     void unableToDeserializeMessage(String topic, @Cause Throwable t);
+
+    @LogMessage(level = Logger.Level.WARN)
+    @Message(id = 18250, value = "The configuration property '%s' is deprecated and is replaced by '%s'.")
+    void deprecatedConfig(String deprecated, String replace);
```

Comment:
I was thinking maybe we could add `@Once` here, because if you have multiple Kafka channels, you will get the warning multiple times. But with `@Once`, we would only get the first warning, and the warnings about other deprecated properties would be lost. So probably best to keep it as is.

Reply:
Nice, I didn't know about `@Once`. It can come in handy in the future. Let's keep it as is.